src/background/background.ts

```diff
@@ -1,4 +1,151 @@
-// TODO: background script
-chrome.runtime.onInstalled.addListener(() => {
-  // TODO: on installed function
-})
+// inputs:
+const cvText = `Eytan Krief
```

Comment:
don't commit your test data

Reply:
What if i actually need this for the code to run properly until we get an input?